drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/main/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/editor/strategies/BusinessCentralDMODataManagementStrategy.java

```diff
@@ -109,15 +115,19 @@ protected void manageDataObjects(List<String> dataObjectsTypes, final TestToolsV
         Callback<FactModelTree> aggregatorCallback = aggregatorCallback(testToolsPresenter, expectedElements, dataObjectsFieldsMap, context, simpleJavaTypes, gridWidget);
         // Iterate over all dataObjects to retrieve their modelfields
         dataObjectsTypes.forEach(factType ->
-                                         oracle.getFieldCompletions(factType, fieldCompletionsCallback(factType, aggregatorCallback)));
-
+                                         oracle.getFieldCompletions(factType, fieldCompletionsCallback(factType, superTypeMap, aggregatorCallback)));
     }
 
     @Override
     protected List<String> getFactTypes() {
         return Arrays.asList(oracle.getFactTypes());
     }
 
+    @Override
+    protected void getSuperType(String factType, Callback<String> callback) {
```

Comment:
Looks like it should return something but it dosen't. Maybe we can avoid naming it getter in this case?